helpers/setup-sa.sh

```diff
@@ -130,4 +157,28 @@ gcloud services enable \
   appengine.googleapis.com \
   --project ${HOST_PROJECT}
 
+# enable the billing account
+if [[ ${BILLING_ACCOUNT:-} != "" ]]; then
+  echo "Enabling the billing account..."
+  gcloud beta billing accounts get-iam-policy $BILLING_ACCOUNT > policy-tmp-$$.yml
+  unamestr=`uname`
+  if [ "$unamestr" = 'Darwin' ]; then
+    sed -i '' -e "/^etag:.*/i \\
+- members:\\
+\ \ - serviceAccount:${SA_ID}\\
+\ \ role: roles/billing.user" policy-tmp-$$.yml
+    gcloud beta billing accounts set-iam-policy $BILLING_ACCOUNT policy-tmp-$$.yml
+  elif [ "$unamestr" = 'Linux' ]; then
+    sed -i '' -e "/^etag:.*/i \\
+- members:\\
+\ \ - serviceAccount:${SA_ID}\\
+\ \ role: roles/billing.user" policy-tmp-$$.yml
+    gcloud beta billing accounts set-iam-policy $BILLING_ACCOUNT policy-tmp-$$.yml
+  else
+      echo "Could not set roles/billing.user on service account $SERVICE_ACCOUNT.\
```

Comment:
Small fixup: since we used a 2 space indent in the previous code, could we switch this block to use 2 spaces as well?